pkg/cmdlets/errors/errors.go

```diff
@@ -113,14 +113,6 @@ func ParseUserFacing(err error) (int, error) {
 		return code, nil
 	}
 
-	// If there is a user facing error in the error stack we want to ensure
-	// that is it forwarded to the user.
-	var userFacingError errs.UserFacingError
-	if errors.As(err, &userFacingError) {
-		logging.Debug("Returning user facing error, error stack: \n%s", errs.JoinMessage(err))
-		return code, &OutputError{userFacingError}
-	}
```

Comment:
This wasn't achieving anything, either way we wrap it in OutputError with the `error` type.